doc/sphinx/source/themes/neo4j/layout.html

```diff
@@ -16,9 +20,9 @@
   <nav class="navbar">
     <div class="navbar-brand">
       <a class="navbar-item" href="/">
-
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20210422142941/neo4j-logo-2020.svg" alt="Neo4j Graph Data Science Client API Reference">
-      </a><a href="/docs/" class="navbar-item no-left-padding" aria-label="Neo4j Graph Data Science Client API Reference">
+        <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20210422142941/neo4j-logo-2020.svg" alt="Neo4j Graph Data Science Client API Reference">
+      </a>
+      <a href="/docs/graph-data-science-client" class="navbar-item no-left-padding" aria-label="Neo4j Graph Data Science Client API Reference">
```

Comment:
Can we target the index of the API refdocs instead? We have the target back to the client manual in the top right already.
```suggestion
      <a href="/docs/graph-data-science-client/{version}/api/" class="navbar-item no-left-padding" aria-label="Neo4j Graph Data Science Client API Reference">
```

This suggestion is probably not good but I don't know how to get the `{version}` part in there.

Reply:
Looks like we can use the `pathto` helper here. I've updated the link.